### ISSUE TITLE
Not Working in 5.5.1 for iOS

### ISSUE THREAD

**Issue by jabele** [Sitata/titanium-google-analytics#60]
Although it is working great in my Android apps I am having trouble getting this to work at all in iOS apps in 5.5.0 or 5.5.1. It is not showing any usage stats at all for iOS despite it being setup the same way as it was in 5.3+ and 5.4+ versions. Anyone else running into these issues?

**Comment by tujoworker** [comment]
I'm getting this logs:

```
[Ti:WARN] Couldn't fetch module analytics.google from HTTP server. Use local file.
[Ti:WARN] Couldn't fetch module analytics.google from file system. Use original require.
```

Do You getting anything like this @jabele ?

**Comment by tujoworker** [comment]
I'm getting this logs:

**Comment by shebby-dogfish** [comment]
Hi @tujoworker or @jabele - did you guys find a fix for this ? I am just seeing the same problem on iOS - Android works fine! Thanks

**Comment by jaceaser** [comment]
I have 2 apps that work with 5.5.1 and one that isn't. I have no clue why 1 isn't working, but I don't see that error in my logs.

**Comment by caiocost** [comment]
You probably have the same problem, look for conflicts on the google_play
jar files of your modules.

Remember that some modules are built in titanium folders

**Comment by jaceaser** [comment]
I don't think that is the issue. I have 2 projects, they have the same module, same SDK, same GA api key. One app will have data in GA, the other won't. I don't see any errors in the logs either.

**Comment by jaceaser** [comment]
I resolved the issue by setting run-on-main-thread to false. My other apps didn't have this setting in the tiapp.xml
